src/@next/Tag/TagStyle.ts

```diff
@@ -74,10 +99,17 @@ export const TagStyle = styled.div<TagProps>`
 
   @media (max-width: ${Breakpoints.large}) {
     font-size: 12px;
+
+    & svg {
+      width: 18px;
+      height: 18px;
+    }
```

Comment:
The size of 18px x 18px for mobile view is not reflected on storybook

Reply:
the 18px x 18px seems correct on my side 🤔

Reply:
![image](https://github.com/glints-dev/glints-aries/assets/67429929/ed842180-4f3c-47a4-aedb-3ecb767442a2)